RAVBA-M/src/win32/MainWnd.cpp

```diff
@@ -571,6 +580,7 @@ bool MainWnd::FileRun()
 		RA_SetConsoleID( 4 );
 		RA_ClearMemoryBanks();
 		RA_InstallMemoryBank( 0, ByteReader, ByteWriter, 0x10000 );
+		RA_InstallMemoryBank( 1, BankedRAMReader, BankedRAMWriter, 0x1000 );
```

Comment:
Banking is specifically a GBC feature, I see no reason to duplicate bank 1 for GB - it would effectively be mirror/echo RAM, which we encourage devs not to use.

Also note that `gbWram` is NULL in GB mode (https://github.com/RetroAchievements/RAEmus/blob/master/RAVBA-M/src/gb/GB.cpp#L2184-L2202), so your changes would cause the emulator to crash if attempting to view this bank.

Reply:
My concern with not mirroring the bank for GB is that for a title that runs in both GB and GBC mode and uses bank-switching in GBC mode, the contents of the bank-switched RAM (0xD000\~0xDFFF) will not necessarily match on both systems, even if bank 1 contents are the same. That said I'm not sure if there *is* a title that is compatible with both models and uses bank-switching on GBC.

Reply:
As mentioned in chat, it seems unlikely for a GB-compatible title to be making use of bank-switching on GBC, though I could not find a definitive answer. For now I have removed the duplicate RAM mapping for GB. If it ends up being needed for cross-platform compatibility, it can be implemented later.